src/PageObject/Element/AdminMenu.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace PaulGibbs\WordpressBehatExtension\PageObject\Element;
+
+use SensioLabs\Behat\PageObjectExtension\PageObject\Element;
+use Behat\Mink\Exception\UnsupportedDriverActionException;
+
+class AdminMenu extends Element
+{
+    protected $selector = '#adminmenu';
+
+    /**
+     * @return array Array of top-level menu item names (i.e. [ "Dashboard", "Posts", ...])
+     */
+    public function getTopLevelMenuItems()
+    {
+        $menuItem_nodes = $this->findAll('css', '#adminmenu > li a .wp-menu-name');
```

Comment:
There may be further instances but for variable names please can we use underscore seperated , not camel case? We're only using camelcase for method names per PSR1. Thank you.